docs/getting_started/install/local/linux.txt

```diff
@@ -251,38 +261,63 @@ and install CrateDB:
     sudo apt-get update
     sudo apt-get install crate
 
-Start CrateDB
--------------
+If you want to install testing releases, you can simply replace
+``ppa:crate/stable`` with ``ppa:crate/testing`` and run ``apt-get update``
+again.
+
+.. _install_from_deb_repo:
+
+Install CrateDB from DEB repository
+-----------------------------------
 
-CrateDB should have been started automatically by upstart and you can stop or
-restart it with:
+.. NOTE::
+
+  This repository contains only CrateDB versions greater than and equal to
+  2.1.0.
+
+First, you need to download and install the GPG key with which the packages
```

Comment:
the line wrapping here is incorrect. perhaps else where in the file?

("are" should be on previous line)